corehq/apps/reports/static/reports/js/bootstrap5/hq_report.js

```diff
@@ -286,7 +286,7 @@ hqDefine("reports/js/bootstrap5/hq_report", [
         function trackReportPageEnter() {
             switch (self.slug) {
                 case 'submit_history':
-                    kissmetrics.track.event('Visited Submit History Page');
+                    kissmetrics.track.event('Visited Submission History Page');
```

Comment:
another flag of kissmetrics event. @kcowger FYI. also comment here if this shouldn't be changed